shared/js/download/download_ui.js

```diff
@@ -132,7 +132,13 @@ var DownloadUI = (function() {
     confirm.setAttribute('data-z-index-level', '100');
 
     var screen = document.getElementById('screen');
-    screen.appendChild(confirm);
+
+    if (screen) {
```

Comment:
nit comment. What about something like:
```javascript
var container = screen || document.body;
container.appendChild(confirm);
```
That way we avoid conditionals, but is up to you.
In any case, we should put the else in the same line than the closing if, just for coherence with the rest of the code base. The code structure is very personal, but that's the agreement we did for gaia in order to maintain the same one.